decision-process/README.md

```diff
@@ -0,0 +1,77 @@
+---
+type: decision
+stakeholders:
+    - sig-architecture
+    - team-horizon
+    - team-team
+issue:
+---
+
+# RFC Decision Making Process
```

Comment:
It would be nice to add how RFC (comments, discussions) are related to ADR (short notes about taken decisions)

Reply:
i'm not sure yet - i have a feeling that RFCs should replace ADRs (because they include more of the backstory), but i could also believe that they're both useful tools for us to have. i kind of want to live with decision rfcs for a bit, and see if they're somehow more useful